doc-src/dev.rst

```diff
@@ -0,0 +1,369 @@
+Contributor's guide
```

Comment:
```suggestion
Contributor's Guide
```

Reply:
Capitalize title properly

Reply:
So "properly" is a bit strong — my understanding is that both "Title Case" and "Sentence case" are correct, and my own preferred style is sentence case (in fact, it's stated as such once you get down to the coding style section). Currently it's the house style for the project, and I'd prefer to keep it that way.